### DISCUSSION TITLE
I keep getting this error when i enable loclist or quickfix

### DISCUSSION THREAD

**Discussion by sorrowlol** [w0rp/ale#4123] [Q&A - Ask for help with problems]
![image](https://user-images.githubusercontent.com/79281626/159964028-f7730e82-bfc1-4b4f-8f49-e3211d5bdd56.png)

The linter works fine now after I made some changes, however I would like to also use the loclist to provide me with list of errors. Whenever I enable it, i get the error in the image. I have to press keys a few times and then i can see the errors in the loclist, but it is annoying as it happens every time. Is there a way around it? 

I have no idea of knowing which function that is as it doesn't say in VIM, I tried using set verbose=1 and still nothing :/

The problem happens when i set:

let g:ale_set_loclist = 1

or 

let g:ale_set_quickfix = 1

Anyone with a similar problem while using ALE Quickfix?

**Comment by sorrowlol** [comment]
I have found the solution. It was NERDTree causing the issue. I had an autocommand on my .vim for NERDTreeMirror and it seemed to cause problems. I removed it from my config file and now everything works perfectly :)

**Reply by pbnj** [reply]
Helpful tip for anyone that might come across this:

When you get an unexpected error in Vim and all you have is a function name to go off of, like 

```
Error detected while processing function <SNR>83_ApplyPartialTimer[3]..<SNR>103_SetListsImpl:
```

One step you can take is `grep` your plugin directory for that function name, like:

```
# if using vim-plug
$ grep -n ApplyPartialTimer -R ${HOME}/.vim/plugged

# if using native vim packages
$ grep -n ApplyPartialTimer -R ${HOME}/.vim/pack/
```

That should help you find the offending plugin and continue investigation from there (e.g. search their GitHub repo for reported issues with that error message).
